exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -206,3 +207,10 @@ def _sanitize(self, key: str) -> str:
         Replace all characters other than [A-Za-z0-9_] with '_'.
         """
         return self._non_letters_digits_underscore_re.sub("_", key)
+
+    # pylint: disable=no-self-use
+    def _check_value(self, value: Union[int, float, str, Sequence]) -> str:
+        """Check the label value and return is appropriate representation"""
+        if not isinstance(value, str) and isinstance(value, Sequence):
+            return dumps(value, default=str)
+        return str(value)
```

Comment:
I re-read the specification. I think the requirement is to json encode the non-string attribute values not just the sequence type. For example `_check_value(True)` returns `'True'` which should actually be `'true'`.